examples/kitchen-sink-example/components/header.tsx

```diff
@@ -1,12 +1,13 @@
+'use client';
 import React from 'react';
 import Link from 'next/link';
-import { useUser } from '@auth0/nextjs-auth0';
+import { useUser } from '@auth0/nextjs-auth0/client';
 
 const Header = (): React.ReactElement => {
   const { user } = useUser();
 
   return (
-    <header>
+    <div id="header">
```

Comment:
Nit: should this be a `<nav>`?

Reply:
It should be a `<header>` - but the Server Component renderer was throwing errors on `<header>`, so I changed it to a `<div>`